packages/wdio-allure-reporter/src/utils.js

```diff
@@ -70,3 +70,20 @@ export const getErrorFromFailedTest = (test) => {
     }
     return test.error
 }
+
+/**
+ * Substitute task id to link template
+ * @param {string} template - link template
+ * @param {string} id - task id
+ * @returns {string} - link after substitution
+ * @private
+ */
+export const getLinkByTemplate = (template, id) => {
+    if (!template || typeof template !== 'string') {
```

Comment:
checking of not being a string is enough, no?
```suggestion
    if (typeof template !== 'string') {
```